app/view_models/auctions_index_view_model.rb

```diff
@@ -1,29 +1,30 @@
-class AuctionsIndexViewModel < Struct.new(:current_user, :auctions_query)
-  def auctions
-    @auctions ||= auctions_query.map { |auction| AuctionViewModel.new(current_user, auction) }
+class AuctionsIndexViewModel
+  attr_reader :auctions, :current_user
+
+  def initialize(auctions:, current_user:)
+    @auctions = auctions.to_a
+    @current_user = current_user
   end
 
   def active_auction_count
-    auctions.count { |i| i.started_at < Time.now && Time.now < i.ended_at }
+    auctions.count { |i| i.started_at < Time.current && Time.current < i.ended_at }
```

Comment:
yeah we could totally add a scope for that. In face, we should. Trying to make as few changes as possible here so didn't change the implementation of the method. Do you think we should do that now or wait?